edx_sga/tests/integration_tests.py

```diff
@@ -32,7 +32,7 @@
 from xblock.fields import ScopeIds
```

Comment:
Nit (subjective): please change the commit's title from "refactor"  to "test". This PR does not touch any user-facing code.